content/administration/maintain/google_oauth.rst

```diff
@@ -0,0 +1,240 @@
+   OAuth.
+
+Setup in Google
+===============
+
+Create a new project
+--------------------
+
+To get started, go to `Google APIs platform
+<https://console.cloud.google.com/apis/credentials/consent>`__ Log in with your *Google Workspace*
+account if you have one, otherwise log in with your personal Gmail account. Choose the email
+address you want to configure in Odoo.
+
+After that, click on :guilabel:`Create Project`, located on the far right of the OAuth consent
+screen. If a project has already been created in this account then the new project option will be
+located on the top left under the :guilabel:`Select a project` drop-down menu.
+
+On the new project screen, rename the :guilabel:`Project` to *Odoo* and browse for the
+:guilabel:`Location`. Set the :guilabel:`Location` as the *Google Workspace* organization. If you
+are using a personal Gmail account, then leave the :guilabel:`Location` as *No Organization*.
+
+.. image:: google_oauth/new-project.png
+   :class: img-fluid
+   :align: center
+   :alt: Project Name and Location for Google Oauth.
+
+Next, click on :guilabel:`Create`, and then the :guilabel:`User Type` screen appears. Select the
+:guilabel:`External` option, and then click on :guilabel:`Create`.
+
+Edit app registration
+---------------------
+
+- On the :guilabel:`Oauth consent screen` step, under the :guilabel:`App information` section,
+  enter *Odoo* in the :guilabel:`App name` field. Select the organization's email address under the
+  :guilabel:`User support` email field.
+
+.. image:: google_oauth/app-name-odoo.png
+   :class: img-fluid
+   :align: center
+   :alt: App Name and User Support Email on Google OAuth.
+
+- Next, under :menuselection:`App Domain --> Authorized domains`, click on :guilabel:`Add Domain` and
+  enter *odoo.com*.
+- After that, under the :guilabel:`Developer contact information` section, enter the organization's
+  email address. Google uses this email address to notify the organization about any changes to
+  your project.
+- Next, click on the :guilabel:`Save and Continue` button.
+- Then, skip the :guilabel:`Scopes` page by scrolling to the bottom and clicking on
+  :guilabel:`Save and Continue`.
+- Next, add the email being configured under the :guilabel:`Test users` step by clicking on
+  :guilabel:`Add Users` and then the :guilabel:`Save and Continue` button. A summary of the
+  :guilabel:`App registration` appears.
+- Finally, scroll to the bottom and click on :guilabel:`Back to Dashboard`.
+
+Create Credentials
+------------------
+
+Now that you have set up the project, you need to create credentials, which includes the
+:guilabel:`Client ID` and :guilabel:`Client Secret`. First, click on :guilabel:`Credentials` in the
+left sidebar menu.
+
+Then, click on :guilabel:`Create Credentials` in the top menu and select
+:guilabel:`OAuth client ID` from the dropdown menu.
+
+.. image:: google_oauth/create-credentials.png
```

Comment:
This image can be deleted.

Reply:
Looks like this one might have been accidentally missed! Please delete Lines 72-76 and delete the image file from the file tree by using `git rm image-file-path`